File: samples/search/python/upload-youtube-link.py
import requests
from requests.auth import HTTPBasicAuth
import json

api_key = ""  # Your API key here, required
youtube_link = ""   # YouTube link here, required
url = "https://api-us.musiio.com/v1/search/upload/youtube-link"

payload = {
    "link": youtube_link
}
headers = {
  'Content-Type': 'application/json'
}

response = requests.post(url, auth=HTTPBasicAuth(api_key, ''), headers=headers, data=json.dumps(payload))

print(response.text.encode('utf8'))

File: samples/search/node/upload-youtube-link.js
const API_KEY = "";     // API KEY here
const YOUTUBE_LINK = "";// Youtube link here

var axios = require('axios');
var data = JSON.stringify({"link": YOUTUBE_LINK});

var config = {
  method: 'post',
  url: 'https://api-us.musiio.com/v1/search/upload/youtube-link',
  auth: {
      username: API_KEY,
      password: ""
  },
  headers: { 
    'Content-Type': 'application/json'
  },
  data : data
};

axios(config)
.then(function (response) {
  console.log(JSON.stringify(response.data));
})
.catch(function (error) {
  console.log(error);
});
